src/components/Content.js

```diff
@@ -29,63 +29,64 @@ import {
   meta,
 } from '../util.js'
 
-const themeLocal = localStorage['Settings/Theme'] || 'Dark'
-const fontSizeLocal = +(localStorage['Settings/Font Size'] || 16)
 const tutorialLocal = localStorage['Settings/Tutorial'] === 'On'
 const tutorialStepLocal = +(localStorage['Settings/Tutorial Step'] || 1)
 
-export const Content = connect(({ dataNonce, focus, search, user, settings, dragInProgress, isLoading, showModal }) => {
-  const dark = (isLoading ? themeLocal : getSetting('Theme')[0]) !== 'Light'
-  const scaleSize = (isLoading ? fontSizeLocal : getSetting('Font Size')[0] || 16) / 16
+const mapStateToProps = ({ focus, search, isLoading, showModal }) => {
   const tutorial = isLoading ? tutorialLocal : meta([EM_TOKEN, 'Settings', 'Tutorial']).On
   const tutorialStep = isLoading ? tutorialStepLocal : getSetting('Tutorial Step')[0] || 1
+  const rootThoughts = getThoughtsRanked(RANKED_ROOT)
   return {
-    dataNonce,
-    dark,
-    dragInProgress,
     focus,
-    isLoading,
-    scaleSize,
     search,
     showModal,
     tutorial,
     tutorialStep,
-    user,
+    rootThoughts
   }
-})((
-  { dataNonce, search, user, dragInProgress, dark, tutorialStep, isLoading, dispatch, showModal, scaleSize }) => {
+}
 
-  const rootThoughts = getThoughtsRanked(RANKED_ROOT)
+const mapDispatchToProps = dispatch => ({
+  showRemindMeLaterModal: () => dispatch({ type: 'modalRemindMeLater', MODAL_CLOSE_DURATION }),
+  cursorBack: () => dispatch(cursorBack)
+})
+
+const stopEventPropagation = e => {
+  // stop propagation to prevent default content onClick (which removes the cursor)
+  e.stopPropagation()
+}
```

Comment:
This seems unnecessary. Let's stick with the inline expression for simplicity.

Reply:
This function doesn't rely on props and gets initialized on every re-render. It's best to define such functions/constants outside the component so that the same reference can be pointed to, instead of allocating new memory every time.